freqtrade/freqtradebot.py

```diff
@@ -200,7 +200,7 @@ def check_for_open_trades(self):
         Notify the user when the bot is stopped
         and there are still open trades active.
         """
-        open_trades = Trade.get_trades([Trade.is_open == 1]).all()
+        open_trades = Trade.get_trades([Trade.is_open == True]).all()
```

Comment:
```suggestion
        open_trades = Trade.get_trades([Trade.is_open.is_(True)]).all()
```

Aside from this ... 
what do you have to do to actually reach this error? I've never seen this, but it seems like a problem that SHOULD appear right away ...

Reply:
Hi, I used psycopg2 to connect to PostgreSQL